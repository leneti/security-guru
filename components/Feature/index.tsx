export { default as Feature } from "./Feature";
export * from "./types";
