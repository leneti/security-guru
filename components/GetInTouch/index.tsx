export { default, type ContactForm } from "./GetInTouch";
