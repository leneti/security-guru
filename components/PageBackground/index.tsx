export { default, type PageBGProps } from "./PageBackground";
