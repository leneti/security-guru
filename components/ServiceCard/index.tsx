export { default as ServiceCard } from "./ServiceCard";
export * from "./types";
