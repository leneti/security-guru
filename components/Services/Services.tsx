import { Grid } from "@mantine/core";
import { ServiceCard } from "@components";
import { serviceData } from "@constants";

export default function Services() {
  return (
    <Grid columns={6} grow>
      {serviceData.map((service) => (
        <Grid.Col sm={6} md={3} lg={2} key={service.title}>
          <ServiceCard {...service} />
        </Grid.Col>
      ))}
    </Grid>
  );
}
