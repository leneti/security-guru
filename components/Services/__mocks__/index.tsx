export function Services() {
  return <div>Mock Services</div>;
}
