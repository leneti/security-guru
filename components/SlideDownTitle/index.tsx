export { default as SlideDownTitle } from "./SlideDownTitle";
export * from "./types";
