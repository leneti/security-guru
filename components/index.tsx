export { default as ContactInfo } from "./ContactInfo";
export { default as Footer } from "./Footer";
export { default as GetInTouch, type ContactForm } from "./GetInTouch";
export { default as Header } from "./Header";
export { default as Hero } from "./Hero";
export { default as Logo } from "./Logo";
export { default as PageTitle } from "./PageTitle";
export { default as PageBackground, type PageBGProps } from "./PageBackground";
export { default as SlideDownTitle } from "./SlideDownTitle";
