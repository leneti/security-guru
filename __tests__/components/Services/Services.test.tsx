import { render } from "@testing-library/react";
import { Services } from "@site/components/Services";

jest.mock("@site/components/ServiceCard");

describe("Services", () => {
  it("renders", () => {
    const { container } = render(<Services />);
    expect(container).toMatchSnapshot();
  });
});
