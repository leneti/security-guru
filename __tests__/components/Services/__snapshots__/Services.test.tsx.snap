// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services renders 1`] = `
<div>
  <div
    class="mantine-Grid-root mantine-16fdnqw"
  >
    <div
      class="mantine-Grid-col mantine-xlsyji"
    >
      <h3>
        Mock title: 
        Apsaugos signalizacijos sistemos
      </h3>
      <div>
        Mock image: 
        {"src":"/img.jpg","height":40,"width":40,"blurDataURL":"data:image/png;base64,imagedata"}
      </div>
      <div>
        Mock description: 
        Montuojame ir įrengiame apsaugos signalizacijas namuose ir biuruose, kad užtikrinti jūsų patalpų saugumą.
      </div>
      <div>
        Mock price: 
        nuo €200
      </div>
      <div>
        Mock url: 
        /paslaugos/apsaugos-signalizacijos-sistemos
      </div>
    </div>
    <div
      class="mantine-Grid-col mantine-xlsyji"
    >
      <h3>
        Mock title: 
        Įeigos kontrolės sistemos
      </h3>
      <div>
        Mock image: 
        {"src":"/img.jpg","height":40,"width":40,"blurDataURL":"data:image/png;base64,imagedata"}
      </div>
      <div>
        Mock description: 
        Diegiame ir įrengiame eismo valdymo sistemas biuruose ir viešosiose vietose eismo srautams valdyti ir saugumui užtikrinti.
      </div>
      <div>
        Mock price: 
        nuo €100
      </div>
      <div>
        Mock url: 
        /paslaugos/ieigos-kontroles-sistemos
      </div>
    </div>
    <div
      class="mantine-Grid-col mantine-xlsyji"
    >
      <h3>
        Mock title: 
        Priešgaisrinės signalizacijos sistemos
      </h3>
      <div>
        Mock image: 
        {"src":"/img.jpg","height":40,"width":40,"blurDataURL":"data:image/png;base64,imagedata"}
      </div>
      <div>
        Mock description: 
        Montuojame ir įrengiame priešgaisrinę signalizaciją namuose ir biuruose, kad aptiktume ir įspėti apie gaisrą.
      </div>
      <div>
        Mock price: 
        nuo €200
      </div>
      <div>
        Mock url: 
        /paslaugos/priesgaisrines-signalizacijos-sistemos
      </div>
    </div>
    <div
      class="mantine-Grid-col mantine-xlsyji"
    >
      <h3>
        Mock title: 
        Vaizdo stebėjimo sistemos
      </h3>
      <div>
        Mock image: 
        {"src":"/img.jpg","height":40,"width":40,"blurDataURL":"data:image/png;base64,imagedata"}
      </div>
      <div>
        Mock description: 
        Montuojame ir įrengiame stebėjimo sistemas namuose ir biuruose, kad  stebėti veiklą ir aptikti bet kokį įtartiną elgesį.
      </div>
      <div>
        Mock price: 
        nuo €200
      </div>
      <div>
        Mock url: 
        /paslaugos/vaizdo-stebejimo-sistemos
      </div>
    </div>
    <div
      class="mantine-Grid-col mantine-xlsyji"
    >
      <h3>
        Mock title: 
        Integruoti apsaugos sprendimai
      </h3>
      <div>
        Mock image: 
        {"src":"/img.jpg","height":40,"width":40,"blurDataURL":"data:image/png;base64,imagedata"}
      </div>
      <div>
        Mock description: 
        Pagal užsakymą įrengiame integruotą apsaugos sistemą, kuri apima apsaugos signalizaciją, eismo kontrolę, priešgaisrinę signalizaciją ir stebėjimo sistemas.
      </div>
      <div>
        Mock price: 
        nuo €200
      </div>
      <div>
        Mock url: 
        /paslaugos/integruoti-apsaugos-sprendimai
      </div>
    </div>
  </div>
</div>
`;
